packages/core/src/table/table.scss

```diff
@@ -0,0 +1,158 @@
+    font-weight: $cds-global-typography-font-weight-bold;
+    border-bottom-width: $cds-alias-object-border-width-100;
+    border-bottom-style: solid;
+    border-bottom-color: $cds-alias-object-border-color;
+    border-top-width: $cds-alias-object-border-width-100;
+    border-top-style: solid;
+    border-top-color: $cds-alias-object-border-color;
+  }
+
+  td {
+    background-color: $cds-alias-table-color-background;
+    font-size: #{$cds-global-typography-font-size-3};
+    border-bottom: $cds-alias-object-border-width-100 solid $cds-alias-object-border-color-tint;
+  }
+
+  tr th:first-child,
+  tr td:first-child {
+    border-left: $cds-alias-object-border-width-100 solid $cds-alias-object-border-color;
+  }
+
+  tr th:last-child,
+  tr td:last-child {
+    border-right: $cds-alias-object-border-width-100 solid $cds-alias-object-border-color;
+  }
+
+  tr:last-child td {
+    border-bottom-color: $cds-alias-object-border-color;
+  }
+
+  tfoot th {
+    border-top-width: 0;
+  }
+}
+
+// Border radius styles
+[cds-table] {
+  thead tr:first-child {
+    :first-child {
+      border-radius: 0;
+      border-top-left-radius: $cds-alias-object-border-radius-50;
+    }
+
+    :last-child {
+      border-radius: 0;
+      border-top-right-radius: $cds-alias-object-border-radius-50;
+    }
+  }
+  & > *:last-child > tr:last-of-type {
+    :first-child {
+      border-radius: 0;
+      border-bottom-left-radius: $cds-alias-object-border-radius-50;
+    }
+
+    :last-child {
+      border-radius: 0;
+      border-bottom-right-radius: $cds-alias-object-border-radius-50;
+    }
+  }
+}
+
+// Left Alignment options
+[cds-table~='left'] th,
+[cds-table~='left'] td,
+[cds-table] td[cds-col~='left'],
+[cds-table] th[cds-col~='left'] {
+  text-align: left;
+}
+[cds-table~='left'] tr th:first-child,
+[cds-table~='left'] tr td:first-child,
+[cds-table] tr td[cds-col~='left']:first-child,
+[cds-table] tr th[cds-col~='left']:first-child {
+  padding-left: #{$cds-global-layout-space-sm};
+}
+
+// Right Alignment options
+[cds-table~='right'] th,
+[cds-table~='right'] td,
+[cds-table] td[cds-col~='right'],
+[cds-table] th[cds-col~='right'] {
+  text-align: right;
+}
+
+// No Border options
+[cds-table~='noborder'],
+[cds-table~='noborder'] tr th,
+[cds-table~='noborder'] tr td {
+  border-width: 0 !important;
+  background-color: transparent;
+}
+[cds-table~='noborder'] tr th {
+  border-bottom-width: $cds-alias-object-border-width-100 !important;
+}
+
+// Compact options
+[cds-table~='compact'] {
+  td,
+  th {
+    line-height: #{$cds-global-typography-font-size-4};
+    padding: #{$cds-global-typography-font-size-1} #{$cds-global-typography-font-size-2};
+    padding-top: #{$cds-global-layout-space-sm};
+    padding-bottom: #{$cds-global-layout-space-xs};
+  }
+}
+
+// Vertical options
+[cds-table~='vertical'] {
+  tr td:first-child,
+  tr th:first-child {
+    background-color: $cds-alias-status-neutral-tint;
+    // color: ??;
```

Comment:
If you use `cds-alias-object-interaction-background-selected` and `cds-alias-object-interaction-color-selected` you will get the right contrast levels for light and dark theme.